modules/@apostrophecms/rich-text-widget/ui/src/index.scss

```diff
@@ -0,0 +1,4 @@
+.bp-rich-text img {
```

Comment:
This is the bare minimum styling for inserted inline images to not blow out the page width.